staff/tachi-fernandez/cv/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curiiculum Vitae</title>
+</head>
+<body>
+  <p>
```

Comment:
The <p> tag has not this use (https://www.w3schools.com/tags/tag_p.asp), you could try to find something more semantic.